src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -84,56 +76,6 @@ def test_check_name_availability(self):
             self.check('nameAvailable', True)
         ])
 
-    @ResourceGroupPreparer()
-    @StorageAccountPreparer(parameter_name='storage_account_for_update')
-    def test_acr_create_with_new_storage(self, resource_group, resource_group_location,
```

Comment:
are you removing the test on purpose? if yes, you probably should remove the recording file as well
//CC: @djyou

Reply:
@yugangw-msft Yes this was intentional. ACR removed support for `Classic` registries in all api versions , thereby also retiring the use of the `--storage-account-name` parameter. The PR https://github.com/Azure/azure-cli/pull/9676/files addresses these changes in the CLI. Thank you. //cc @ankurkhemani